### PR TITLE
Shareable link

### PR DESCRIPTION
**Pull request by OAGr**

### CODE REVIEW

File: src/lib/guesstimate_api/resources/Models.js [hunk 1]
```diff
@@ -8,10 +8,14 @@ export default class Models extends AbstractResource {
     this.guesstimateMethod({url, method})(callback)
   }
 
-  get(msg, callback) {
-    const url = `spaces/${msg.spaceId}`
+  get(spaceId, token, callback) {
+    let url = `spaces/${spaceId}`
     const method = 'GET'
 
+    // TODO(Ozzie): Offhand, do you know of a better way to do this? Setting it as data does *not* work as this is a
```
Comment (OAGr) [inline src/lib/guesstimate_api/resources/Models.js]: i need to get to this

File: src/lib/guesstimate_api/resources/Models.js [hunk 2]
```diff
@@ -8,11 +8,13 @@ export default class Models extends AbstractResource {
     this.guesstimateMethod({url, method})(callback)
   }
 
-  get(msg, callback) {
-    const url = `spaces/${msg.spaceId}`
+  get(spaceId, token, callback) {
```
Comment (OAGr) [inline src/lib/guesstimate_api/resources/Models.js]: make shareable link

File: src/components/spaces/show/header.js [hunk 3]
```diff
@@ -65,11 +105,24 @@ export class SpaceHeader extends Component {
...
+
+            {isPrivate && editableByMe &&
+              <ShareableLinkOption
+                shareableLinkUrl={shareableLinkUrl}
+                onEnable={onEnableShareableLink}
+                onDisable={onDisableShareableLink}
+                onRotate={onRotateShareableLink}
+              >
+                <div className='ui image label' >
+                  <Icon name='link'/> Link
```
Comment (mmcdermott) [inline src/components/spaces/show/header.js]: Link is probably a bit confusing. Maybe 'Shareable Link'? But your call either way.

File: src/components/utility/card/index.js [hunk 4]
```diff
@@ -11,6 +11,16 @@ String.prototype.capitalizeFirstLetter = function() {
     return this.charAt(0).toUpperCase() + this.slice(1);
 }
 
+export class CardListSection extends Component {
```
Comment (mmcdermott) [inline src/components/utility/card/index.js]: Is this necessary? If so, can it be made a functional component?

File: src/components/spaces/show/header.js (hunk 3 above)
Comment (OAGr) [inline src/components/spaces/show/header.js]: I think this will work; Shareable link would be nice, but we are low on space here.

**Pull request merged**